modules/client-configuration/pages/supported-features-sles.adoc

```diff
@@ -49,8 +49,8 @@ The icons in this table indicate:
 | CVE auditing |  icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
 | SCAP auditing |   icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
 | Package verification | ``Traditional`` | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Package locking |  ``Traditional`` | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| System locking |  ``Traditional`` | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Package locking |  ``Traditional`` | ``Traditional`` | ``Traditional`` | icon:check[role="green"]
```

Comment:
Only for traditional stack available. I am unsure if we ship it for openSUSE 15.1. I think not. So it is not available.